static/css/documentation.css

```diff
@@ -1347,6 +1347,39 @@ p code {
   }
 }
 
+.request-button {
+  cursor: pointer;
+  background-color: #297fc7;
+  color: #fff;
+  font-size: 16px;
+  padding: 15px;
+  border-radius: 5px;
+  border-width: 0px;
+}
+
+.request-button:hover:not([disabled]) {
+  text-decoration: none;
+  background-color: #3088d0;
+}
+
+.request-button:disabled,
+.request-button[disabled] {
```

Comment:
Do we need both of these? And can't we put them on one line if so?

Reply:
I think the second one is for compatibility with older browsers. I moved them to one line: 72eb9ab2535d7485d7fad45deb43be78db6c8c6a